src/nih_wayfinding/app/scripts/views/navigate/navigate-controller.js

```diff
@@ -146,25 +158,23 @@
             var turnIcon = Directions.getTurnIconName(position.properties.directions.turn);
             var speedMs = (ProfileService.getCurrentUser().preferences.speed || 1) * mphToMs;
             var timeMins = (distanceToTurn / speedMs / 60).toFixed(0);
-            var distanceText = 'In approx. ' +  timeMins +
-                ' min (' + $filter('distance')(distanceToTurn) + ')';
             if (position.properties.turnamenity) {
-                distanceText += ' at the ' + position.properties.turnamenity.name;
+                text += ' at the ' + position.properties.turnamenity.name;
```

Comment:
Small thing:  I've noticed that if the place name is just the name of the intersection, the 'the' is out of place.  Not sure if there's an easy fix for that.